### PR TITLE
Add groupped navigation

### PR DESCRIPTION
**Pull request by karelhala**
This is preparation to add landing page application. If no cloud services echnology is selected (e.g. not clicked from landing page) legacy sring will be shown with home button and such.

### Before
screenshot from 2019-03-06 16-31-00

### After
screenshot from 2019-03-06 16-30-23

### CODE REVIEW

File: src/sass/partials/_sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+aside.pf-c-page__sidebar {
+    padding-top: 0px;
+
+    .ins-c-navigation__header {
+        max-height: 4.4rem;
```
Comment (ryelo) [inline src/sass/partials/_sidebar.scss]: According to the mocks, this should be the same height as the page header. Also, we need to use the rem utility here

File: src/js/App/Sidenav/index.js [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
-export default () => <Navigation />;
+class SideNav extends Component {
+    onHomeButtonClick() {
+        window.location.href = `${document.baseURI}platform/landing`;
+    }
+
+    render() {
+        return (<Fragment>
+            <Split className="ins-c-navigation__header">
+                <SplitItem onClick={this.onHomeButtonClick}><HomeIcon size="md" /></SplitItem>
```
Comment (ryelo) [inline src/js/App/Sidenav/index.js]: We should either use an `a` or a `button` here for accessibility reasons

File: src/sass/partials/_sidebar.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        overflow: hidden;
+
+        .pf-l-split__item {
+            padding: 20px;
+            border-bottom: var(--pf-global--BorderWidth--sm) solid #cccccc;
+            &:first-child {
+                cursor: pointer;
```
Comment (ryelo) [inline src/sass/partials/_sidebar.scss]: We shouldn't need this when the home button is converted to an `a` or a `button`

File: src/sass/partials/_sidebar.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+aside.pf-c-page__sidebar {
+    padding-top: 0px;
+
+    .ins-c-navigation__header {
+        max-height: 4.4rem;
+        overflow: hidden;
+
+        .pf-l-split__item {
+            padding: 20px;
```
Comment (ryelo) [inline src/sass/partials/_sidebar.scss]: Please use the rem Utility

File: src/sass/partials/_sidebar.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
+aside.pf-c-page__sidebar {
+    padding-top: 0px;
```
Comment (ryelo) [inline src/sass/partials/_sidebar.scss]: `padding-top: 0` should suffice. No need to put `px` afterwards

File: src/sass/partials/_sidebar.scss [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    .ins-c-navigation__header {
+        max-height: 4.4rem;
+        overflow: hidden;
+
+        .pf-l-split__item {
+            padding: 20px;
+            border-bottom: var(--pf-global--BorderWidth--sm) solid #cccccc;
+            &:first-child {
+                cursor: pointer;
+                border-right: var(--pf-global--BorderWidth--sm) solid #cccccc;
```
Comment (ryelo) [inline src/sass/partials/_sidebar.scss]: Could we use a PF color here?

File: src/sass/partials/_footer.scss [hunk 7]
```diff
@@ -1,5 +1,5 @@
 .ins-c-footer {
-  background: #1d1d1d;
+  background: var(--pf-global--BackgroundColor--300);
   color: #6c6c6c;
```
Comment (ryelo) [inline src/sass/partials/_footer.scss]: Is there any reason we wouldn't want to use a PF color here?

File: src/sass/partials/_sidebar.scss (hunk 1 above)
Comment (karelhala) [reply src/sass/partials/_sidebar.scss]: Good point!

File: src/sass/partials/_sidebar.scss (hunk 4 above)
Comment (karelhala) [reply src/sass/partials/_sidebar.scss]: Will do.

File: src/js/App/Sidenav/index.js (hunk 2 above)
Comment (karelhala) [reply src/js/App/Sidenav/index.js]: Oh yeah, good point.

**Comment by iphands** [comment]
Using admin privs to squash and merge. I think we would like to update the things @ryelo called out... but I'd like to circle back and do it. We need to get the cloud CI working asap.

**Pull request merged**